STAR/draft-ietf-acme-star.md

```diff
@@ -193,21 +192,18 @@ For a more generic treatment of STAR certificates, readers are referred to {{I-D
 ### Name Delegation Use Case
 
 The proposed mechanism can be used as a building block of an efficient name-delegation protocol, for example one that exists between a CDN or a cloud provider and its customers {{I-D.sheffer-acme-star-request}}.  At any time, the service customer (i.e., the IdO) can terminate the delegation by simply instructing the CA to stop the automatic renewal and letting the currently active certificate expire shortly thereafter.
+Note that in this case the delegated entity needs to access the auto-renewed
+certificate without being in possession of the ACME account key that was used
+for initiating the STAR issuance (see also {{get-cert}}).
 
 ## Terminology
 
 IdO
 : Identifier Owner, the owner of an identifier, e.g.: a domain name, a telephone number.
 
-DNO
-: Domain Name Owner, a type of IdO whose identifier is a domain name.
-
 STAR
 : Short-Term and Automatically Renewed X.509 certificates.
```

Comment:
The term NDC was removed, but it's still use in the Implementation Status section.

Reply:
Right.  Since the implementation status section will go away eventually, I'm going to make add the acronym expansion directly in that section.